### PR TITLE
fix(wording): updates

### PR DESCRIPTION
**Pull request by nstuyvesant**
Found a few small English grammatical errors and thought I would submit the corrections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,36 +230,36 @@ export interface PluginOptions {
...
+   * @remarks Before 1.7, default was true
    */
   skipDiagnostics?: boolean
 
   /**
-   * Customize typescript lib folder path
+   * Customize Typescript lib folder path
    *
-   * Should pass a relative path to root or a absolute path
+   * SRelative path to root or an absolute path
```
Comment (qmhc) [inline README.md:262]: Did the `S` miss to be removed?

**Review by qmhc — CHANGES_REQUESTED** [review]
Thanks for your proofreading and I learned a lot.

File: README.md (hunk 1 above)
Comment (nstuyvesant) [reply README.md:262]: Looks like a typo on my part! The first "S" should be removed.

Comment (nstuyvesant) [reply README.md]: Nice catch - I just pushed an update to fix that.

**Review by qmhc — APPROVED** [review]

**Pull request merged**